SCClassLibrary/JITLib/ProxySpace/extStoreOn.sc

```diff
@@ -326,7 +334,11 @@
 			keys = this.monitors.collect { |item| item.key(envir) };
 		};
 		str = String.streamContents { |stream|
-			stream << "// ( p = ProxySpace.new(s).push; ) \n\n";
+			if (Ndef.all.includes(this)) {
```

Comment:
this could also be a message of `Ndef`, couldn't it?

Reply:
yes, probably, though, something like
```Ndef.includes(~space)``` ? 
is not much clearer than what is there - not sure if its worth it.